pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -600,6 +601,12 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		}
 	}
 
+	if vmi.Spec.Domain.Bootloader != nil {
+		if (*vmi.Spec.Domain.Bootloader) == v1.BootloaderEFI {
+			domain.Spec.OS.Loader.Path = EFIBootloaderPath
```

Comment:
> For instance, if the loader path points to an UEFI image, type should be pflash.

From the [docs](https://libvirt.org/formatdomain.html#elementsOSBIOS), type should also be set to `pflash`.

Reply:
Thanks, I changed it so that the `type` can be passed into the spec as well.